ci/scripts/run_unittest.sh

```diff
@@ -98,6 +98,10 @@ fi
 
 # run unittest
 for test in `ls ${UNITTEST_DIR}`; do
+    if [[ ${test} == *".log" ]] || [[ ${test} == *".info" ]]; then
```

Comment:
why need this ?

Reply:
For some reason there may be log files under unittest foler